Sileo/Sileo/fr.lproj/Categories.strings

```diff
@@ -9,7 +9,7 @@
 */
 
 "No_Category" = "(Aucune Catégorie)";
-"Tweaks" = "Modifications";
+"Tweaks" = "Tweaks";
```

Comment:
I understand why you want to do that change, but it's better to keep it in french.
1 other traduction also translated that one
2 better translate every words in french